### PR TITLE
{math}[foss/2021a] biogeme v3.1.2, biogeme v3.2.8 w/ Python 3.9.5

### PR DESCRIPTION
**Pull request by benczaja**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/b/biogeme/biogeme-3.1.2-foss-2021a-Python-3.9.5.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+easyblock = 'PythonBundle'
+
+name = 'biogeme'
+version = '3.1.2'
+versionsuffix = '-Python-%(pyver)s'
```
Comment (verdurin) [inline easybuild/easyconfigs/b/biogeme/biogeme-3.1.2-foss-2021a-Python-3.9.5.eb:5]: @benczaja We don't use the `Python` `versionsuffix` anymore in this toolchain, so please remove from the `easyconfig` and from the filename.

File: easybuild/easyconfigs/b/biogeme/biogeme-3.1.2-foss-2021a-Python-3.9.5.eb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        'checksums': ['28aecfa3701c293e3b3c6bcab68322d77d1721e10b18d987f592651333f1e438'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/biogeme'],
+}
+
+moduleclass = 'math'
```
Comment (verdurin) [inline easybuild/easyconfigs/b/biogeme/biogeme-3.1.2-foss-2021a-Python-3.9.5.eb:36]: Please add a newline at the end of the file.

File: easybuild/easyconfigs/b/biogeme/biogeme-3.2.8-foss-2021a-Python-3.9.5.eb [hunk 3]
```diff
@@ -0,0 +1,39 @@
+easyblock = 'PythonBundle'
+
+name = 'biogeme'
+version = '3.2.8'
+versionsuffix = '-Python-%(pyver)s'
```
Comment (verdurin) [inline easybuild/easyconfigs/b/biogeme/biogeme-3.2.8-foss-2021a-Python-3.9.5.eb:5]: Same here - please remove the `versionsuffix`.

File: easybuild/easyconfigs/b/biogeme/biogeme-3.2.8-foss-2021a-Python-3.9.5.eb [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        'checksums': ['eafb969bb086a014617fae92e652fb54dd98ac72516a360e40c837244935859c'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/biogeme'],
+}
+
+moduleclass = 'math'
```
Comment (verdurin) [inline easybuild/easyconfigs/b/biogeme/biogeme-3.2.8-foss-2021a-Python-3.9.5.eb:39]: Please add a newline.

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/b/biogeme/biogeme-3.1.2-foss-2021a-Python-3.9.5.eb (hunk 2 above)
Comment (benczaja) [reply easybuild/easyconfigs/b/biogeme/biogeme-3.1.2-foss-2021a-Python-3.9.5.eb:36]: @verdurin newline added, thanks for pointing this out.

File: easybuild/easyconfigs/b/biogeme/biogeme-3.1.2-foss-2021a-Python-3.9.5.eb (hunk 1 above)
Comment (benczaja) [reply easybuild/easyconfigs/b/biogeme/biogeme-3.1.2-foss-2021a-Python-3.9.5.eb:5]: @verdurin `versionsuffix` removed from both config and filename. Thanks!

File: easybuild/easyconfigs/b/biogeme/biogeme-3.2.8-foss-2021a-Python-3.9.5.eb (hunk 4 above)
Comment (benczaja) [reply easybuild/easyconfigs/b/biogeme/biogeme-3.2.8-foss-2021a-Python-3.9.5.eb:39]: @verdurin newline added

File: easybuild/easyconfigs/b/biogeme/biogeme-3.2.8-foss-2021a-Python-3.9.5.eb (hunk 3 above)
Comment (benczaja) [reply easybuild/easyconfigs/b/biogeme/biogeme-3.2.8-foss-2021a-Python-3.9.5.eb:5]: @verdurin removed!

**Review by verdurin — DISMISSED** [review]
Looks fine.

**Comment by verdurin** [comment]
@benczaja - thanks. Just a note that I needed to delete the files with the obsolete `versionsuffix` in the filename. They aren't removed automatically when you provide the corrected versions.

**Review by verdurin — APPROVED** [review]

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/344839785a506e5a612977e3c1531c5b for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.5
See https://gist.github.com/7294ac96942f51cd476d54d18a913e47 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/6a25b5e3451c4fecbfea08fdd9539038 for a full test report.

**Comment by verdurin** [comment]
Going in, thanks @benczaja!

**Pull request merged**